operator/pkg/manifest/installer.go

```diff
@@ -114,7 +114,11 @@ var (
 			name.CNIComponentName,
 			name.IngressComponentName,
 			name.EgressComponentName,
-			name.AddonComponentName,
+			name.PrometheusComponentName,
+			name.GrafanaComponentName,
+			name.KialiComponentName,
+			name.TracingComponentName,
+			name.CoreDNSComponentName,
```

Comment:
these shouldn't be needed anymore?

Reply:
> overall i really like this approach. i assume we're going to stick with putting all the addon charts under a single subdir? the name of the dir could be the same as the defined component name.

Yes, that was my plan; the `istio-addon=name` was really just a stop-gap solution to demonstrate this, I now prefer just using the cleaner approach of just the keyword `istio-addon` and the chartName.

> these shouldn't be needed anymore?

Will rebase tomorrow and remove them.